java/README.md

```diff
@@ -0,0 +1,32 @@
+# API Example
+
+Example app exercising some of Climate's [FieldView Platform APIs](https://dev.fieldview.com).
+
+## Setup
+
+1. Install [Java 8+](https://www.java.com/en/download/)
+2. Install [Gradle](https://gradle.org/install/)
+3. Set the following environment variables
+
+```bash
+export CLIENT_ID="my-api-id"
+export CLIENT_SECRET="azbq56fpadhnt8oukoeani2a4w"
+export API_KEY="my-api-id-216b9875-0158-4142-1ab2-7c3bdbd6a2157"
+export API_SCOPES=" fields:read asPlanted:read"
```

Comment:
Is the leading space intentional?

Reply:
leading space? I do see any I am using eclipse editor.